rofl-containers/Cargo.toml

```diff
@@ -11,7 +11,15 @@ rofl-appd = { path = "../rofl-appd" }
 # Third party.
 anyhow = "1.0.86"
 base64 = "0.22.1"
+byteorder = "1.5.0"
 cmd_lib = "1.9.5"
 hex = "0.4.3"
+libc = "0.2.172"
 nix = { version = "0.29.0", features = ["signal"] }
```

Comment:
Isn't `ioctl` feature needed to get the `ioctl_` macros? (e.g. https://docs.rs/nix/0.29.0/nix/macro.ioctl_read.html)

Reply:
Maybe the resolver pulls the `ioctl` feature from some other dependency. Otherwise this indeed could not work (but currently it does even without the explicit `ioctl` feature). I'll add the feature here just to be sure.